src/main/resources/application.properties

```diff
@@ -59,11 +61,11 @@ auth.publicEndpoints=/,/favicon.ico,/oauth/start,/oauth/callback
 # Default Database Configuration
 vcap.services.pz-postgres.credentials.jdbc_uri=jdbc:postgresql://localhost:5432/piazza
 # User used for all database read/write interactions
-vcap.services.pz-postgres.credentials.username=piazza
-vcap.services.pz-postgres.credentials.password=piazza
+vcap.services.pz-postgres.credentials.username=postgres
```

Comment:
Let's remove these changes from the PR.  They're all debug values and serve no real purpose other than placeholders. I'd rather leave them as-is.